### PR TITLE
Add Prerequisites that new provider package should fulfils

### PR DESCRIPTION
**Pull request by TobKed**
Related to: [Add list of prerequisites that new provider package should fulfill](https://github.com/apache/airflow/issues/10386)

- [x] directory structure (possibly automated templates or script to create them)
- [ ] usage scenarios for connections in hooks (see discussion in #12128)
- [ ] Information about what is required:
- [ ] hooks
  - [ ] operators/sensors/transfers
  - [ ] unit tests with mocks
  - [ ] integration tests if applicable
  - [ ] example dags for examples and their use (examples, howtos, system tests)
  - [ ] howto guides
- [ ] relation with backport providers.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: docs/apache-airflow-providers/index.rst [hunk 1]
```diff
@@ -224,6 +224,64 @@ Apache projects. As a community, we provide all the software for free and this w
...
+* Add documentation in ``docs/apache-airflow-providers-some-provider/index.rst``
+
+* Update files:
+
+  * ``airflow/CONTRIBUTING.rst``:
+
+    * extras
+    * provider packages
+
+  * ``airflow/providers/dependencies.json``
```
Comment (potiuk) [inline docs/apache-airflow-providers/index.rst]: This should happen automatically by pre-commit.

**Review by potiuk — APPROVED** [review]
One small comment only, but good to go if corrected.

File: docs/apache-airflow-providers/index.rst (hunk 1 above)
Comment (TobKed) [reply docs/apache-airflow-providers/index.rst]: Updated in https://github.com/apache/airflow/pull/13193/commits/0e966708f16816f8d1215efd29e1b355fdc31582

**Comment by TobKed** [comment]
Thanks @potiuk! I rebased nad made small update.

**Review by potiuk — APPROVED** [review]

**Pull request closed without merging**